renv.lock

```diff
@@ -1370,32 +1429,6 @@
       "Hash": "1de7ab598047a87bba48434ba35d497d",
       "Requirements": []
     },
-    "rrtools": {
-      "Package": "rrtools",
-      "Version": "0.1.5",
-      "Source": "GitHub",
-      "RemoteType": "github",
-      "RemoteHost": "api.github.com",
-      "RemoteRepo": "rrtools",
-      "RemoteUsername": "benmarwick",
-      "RemoteRef": "HEAD",
-      "RemoteSha": "acbae5ee9f9ad099ffa645b2b64a6966e40933b7",
-      "Hash": "99234187afbbb50770a4c8d28b3e5f32",
-      "Requirements": [
-        "clisymbols",
-        "crayon",
-        "devtools",
-        "git2r",
-        "glue",
-        "here",
-        "httr",
-        "jsonlite",
-        "rmarkdown",
-        "rstudioapi",
-        "usethis",
-        "whisker"
-      ]
-    },
```

Comment:
I think we keep pushing this package in and out iteratively. I will try to come up with  a sustainable solution